Cargo.toml

```diff
@@ -708,6 +708,7 @@ sinks-logs = [
   "sinks-new_relic",
   "sinks-papertrail",
   "sinks-pulsar",
+  "sinks-openobserve",
```

Comment:
Hi @chaitanya-sistla, thank you for pointing me to [this](https://github.com/vectordotdev/vector/issues/21514#issuecomment-2416884959). That proposal makes sense to me. 

Since there is no precedent, let's think about it a bit:

* wrap an existing sink, hardcode configuration specific to OpenObserve and expose it as a new sink
* add a guide to the existing http docs, see example [here](https://vector.dev/docs/reference/configuration/sinks/kafka/#azure-event-hubs)
* add community guide on how to use it


Overall, I am happy with both options. But if we go with (1), we might need to do a few iterations since there is no precedent. Option (2) is more straightforward to do. For option (3), the guides live in `website/content/en/guides`.

Also, I would like to cc @jszwedko for awareness.

Reply:
Thank you @pront I would like to follow the first approach if that is ideal. Does my PR reflect the Option (1)? Please let me know if I have to make any changes.

Reply:
I think there is precedent for (1). An example would be the recent changes to the Axiom sink: https://github.com/vectordotdev/vector/pull/21362

Reply:
I did sign the CLA but for some reason it is not updated here.
<img width="1163" alt="image" src="https://github.com/user-attachments/assets/03ab73d1-2528-487d-b628-384e92264fd9">

Reply:
> I think there is precedent for (1). An example would be the recent changes to the Axiom sink: https://github.com/vectordotdev/vector/pull/21362

Very neat. @chaitanya-sistla you can take a look at the above PR to get inspiration for option (1).

Reply:
Hello @pront  thank you. Yes I did and made the file changes, does this relate? I would love to make changes in addition but since we don't need any changes in type, I just modified the needed files.

Reply:
@jszwedko could you help me unblock here? Is there anything else in addition that I need to do? Also I did sign the CLA but it appears to be the other way here.

Reply:
Regarding the CLA, can you try from a different browser or cleaning your cookies? Seems like the Github signin failed for you.

Reply:
Thanks, I opened in other browsers, it doesn't show an option called Agree anywhere and auto redirects me to github once I signed in with my github.

Reply:
> Chaitanya Sistla seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

I think the issue might be that your commits don't have an email address associated. Could you set an email address via `git config user.email` and the use `git rebase` to squash and then `git commit --amend --reset-author` to add an email address to the commits? The email you use should be associated with your GitHub account.

Reply:
> Hello @pront thank you. Yes I did and made the file changes, does this relate? I would love to make changes in addition but since we don't need any changes in type, I just modified the needed files.

The changes here only modify documentation. We need to actually implement another sink to follow existing precedent (i.e. write Rust code). See the `axiom` sink for an example: https://github.com/vectordotdev/vector/blob/master/src/sinks/axiom.rs. That sink also just wraps the `http` sink as you would do. Let me know if that is unclear!

Reply:
Thank you, the cla issue is now resolved. Let me look into this.